src/node_http2.cc

```diff
@@ -2025,7 +2025,11 @@ void Http2Session::OnStreamRead(ssize_t nread, const uv_buf_t& buf_) {
 
   if (LIKELY(stream_buf_offset_ == 0)) {
```

Comment:
```suggestion
  if (LIKELY(stream_buf_offset_ == 0 && static_cast<size_t>(nread) != bs->ByteLength())) {
```

I am not sure if it's likely or unlikely that the buffer needs to be resized, just guessing here.